packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -146,29 +140,6 @@ export const prepareVaultDirector = (
     });
   };
 
-  /**
-   *
-   * @param {VaultManager} vaultManager
-   * @param {Installation<unknown>} oldInstall
-   * @param {unknown} oldTerms
-   */
-  const watchGovernance = (vaultManager, oldInstall, oldTerms) => {
```

Comment:
I guess you know where to look if you want this back when we add liquidation via auction?

Reply:
The vaultManagers were responsible for starting the liquidation contract, but the Auction runs independently of the Vaults. There won't be a need to bring this back.